test/testModel.cc

```diff
@@ -445,7 +446,7 @@ SUITE(Canvas)
       cv->sliderMax=2000;
       // work out where slider is located
       RenderVariable rv(*cv);
-      float xc=cv->x()+rv.handlePos(), yc=cv->y()-rv.height();
+      float xc=cv->x()+rv.handlePos(), yc=cv->y();
```

Comment:
I don't get this How can the mouse be on the slider when it is in the middle of the variable?

Reply:
I've managed to fix this, so that's good.